src/components/TodoList/TodoList.tsx

```diff
@@ -2,223 +1,92 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../../app/store';
+import { Todo } from '../../types/Todo';
+import { selectTodo } from '../../features/currentTodo';
```

Comment:
Ensure that the 'selectTodo' action is correctly implemented in the '../../features/currentTodo' file and is compatible with the expected state updates.